### ISSUE TITLE
Azure RedHat Openshift Support

### ISSUE THREAD

**Issue by marcelloraffaele** [nginxinc/nginx-service-mesh#36]
Hi Team,
It would be great to have support to run NGINX Service Mesh on Azure RedHat Openshift (4.8). 
The documentation talks about "NGINX Service Mesh supports OpenShift version 4.8" but installing the mesh using the following command:
`nginx-meshctl deploy --environment openshift --disable-auto-inject`
 
We are obtaining the following error:

```
Deploying NGINX Service Mesh...
All resources created. Testing the connection to the Service Mesh API Server...
Error: Connection to NGINX Service Mesh API Server failed.
        Check the logs of the nginx-mesh-api container in namespace nginx-mesh for more details.: failed to connect to Mesh API Server, ensure you are authorized and can access services in your Kubernetes cluster; the following events occurred:
        - MountVolume.SetUp failed for volume "spire-agent-socket" : kubernetes.io/csi: mounter.SetUpAt failed to get CSI client: driver name wlapi-mounter.spire.nginx.com not found in the list of registered CSI drivers
        - MountVolume.SetUp failed for volume "spire-agent-socket" : kubernetes.io/csi: mounter.SetUpAt failed to get CSI client: driver name wlapi-mounter.spire.nginx.com not found in the list of registered CSI drivers
        ...
```
 
The situation is the following: 
```
PS> kubectl get pod -n nginx-mesh
NAME                                  READY   STATUS     RESTARTS   AGE
grafana-7c6c88b959-r6qz5              1/1     Running    0          13m
jaeger-86b56bf686-2qq5l               1/1     Running    0          13m
nats-server-5dcd55f8cc-tbmtt          0/2     Init:0/1   0          13m
nginx-mesh-api-6fc8fd675c-8kj2x       0/1     Running    4          13m
nginx-mesh-metrics-7c76545c56-5k7vk   0/1     Running    2          13m
prometheus-8d5fb5879-j2qdb            1/1     Running    0          13m
spire-agent-5ll8h                     3/3     Running    0          13m
spire-agent-8xqwm                     3/3     Running    0          13m
...
spire-server-0                        2/2     Running    0          13m
```

Do you have any suggestion on how to install it on Azure RedHat Openshift?

Thanks

**Comment by jbyers19** [comment]
Hey @marcelloraffaele, our initial RHOS support is limited to self-managed clusters. However, we're in the process of adding support for Azure RHOS now. Once it is ready, we will update this issue.

**Comment by marcelloraffaele** [comment]
Does version 1.4.0 support also Azure RedHat Openshift ?

**Comment by sjberman** [comment]
Should be fixed with release 2.0, closing.
